recipes/parquet-python/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "parquet-python" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/parquet/parquet-{{ version }}.tar.gz
+  sha256: fb1c90768c1b9159d4d6a9b3112ea8107b0b46d7491c5ac452ba7350f333bf0a
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+outputs:
+  - name: parquet-python
+    requirements:
+      host:
+        - python >=3.6
```

Comment:
Where does it say that the minimum version is 3.6? Might be good all the way back to 3.0.

Reply:
conda-forge only builds 3.7+ so ive been turning off <=3.6 for everything as a default. But since this is supposed to be `noarch: python` i'll remove